src/lib.rs

```diff
@@ -1,3 +1,66 @@
-#[test]
-fn it_works() {
+mod test;
+
+use std::collections::HashMap;
+
+/// A structure containing all the common attributes
+/// of the basic OSM data types
+/// 
+/// See: [OSM wiki - Common attributes](https://wiki.openstreetmap.org/wiki/Elements#Common_attributes)
+pub struct ElementInfo {
+    pub id: i64,
+    pub user: String,
+    pub uid: i32,
+    pub timestamp: String,
```

Comment:
Better to use a dedicated object as a chrono::DateTime: more compact and feature full.